views/email/email_partials/breach_alert.hbs

```diff
@@ -0,0 +1,14 @@
+    <tr>
+      <td valign="top" style="padding-top: 3px; padding-bottom: 10px; ">
+        <img src="{{ SERVER_URL }}/img/logos/{{ breachAlert.Name }}.png" width="40" style="width: 40px;" />
```

Comment:
Did we rename all the svg, gif, and jpeg logos to pngs?

Reply:
yeah, but only for emails. I will them pile them all up in a separate PR tomorrow morning.